packages/desktop/electron/lib/menu.js

```diff
@@ -1,6 +1,6 @@
 import { app, ipcMain, Menu, shell } from 'electron'
-import { DISCORD_URL, DOCUMENTATION_URL, FAQ_URL, ISSUE_REPORT_URL } from 'shared/lib/contexts/settings/constants'
-import features from 'shared/features/features'
+import { DISCORD_URL, DOCUMENTATION_URL, FAQ_URL, ISSUE_REPORT_URL } from '@contexts/settings/constants'
+import features from '@features/features'
```

Comment:
Do these work? I'm not sure how these are working in JS if they're specified for TS.

Reply:
I've tested and is working, maybe because of webpack resolve.alias configs

Reply:
Ah yes, probably that :+1: